tokio-trace/src/span.rs

```diff
@@ -0,0 +1,540 @@
+use super::{DebugFields, Dispatcher, StaticMeta, Subscriber, Value};
+use std::{
+    cell::RefCell,
+    cmp, fmt,
+    hash::{Hash, Hasher},
+    ops::Deref,
+    slice,
+    sync::{
+        atomic::{AtomicUsize, Ordering},
+        Arc,
+    },
+    time::Instant,
+};
+
+lazy_static! {
+    static ref ROOT_SPAN: Span = Span::new(
```

Comment:
Longer term nit—I'm not sure there should be a default `ROOT_SPAN`. Might be nice, in the future, to have a request-scoped root Span, and I'm thinking a `lazy_static` might make that tricky to change.

Reply:
yeah, I think we agreed previously that the implicit root span needs to go away... I'll work on that next.

Reply:
Yep, addressed in #4